### ISSUE TITLE
more ugly fix required

### ISSUE THREAD

**Issue by vsrawat** [roger21/imagezoomuglyfixes#8]
further ugly fix required because it is not working with tb 60* and tb 68*

**Comment by roger21** [comment]
yes, this add-on was implicitly dropped when firefox switched to webextensions and didn't support legacy extensions anymore which is since firefox 57, firefox esr 60 and so thunderbird 60
i'm sorry this project is dead, i'll update the main page

**Comment by vsrawat** [comment]
It was very convenient add on, it worked both on ff and tb.

Once it was resurrected from dead.

Hope someone will revive it again. Please help.

Thanks.
-- 
Rawat
